spikeinterface/core/unitsaggregationsorting.py

```diff
@@ -60,25 +61,36 @@ def __init__(self, sorting_list, renamed_unit_ids=None):
             if np.all(annotations == annotations[0]):
                 self.set_annotation(annotation_name, sorting_list[0].get_annotation(annotation_name))
 
-        property_keys = sorting_list[0].get_property_keys()
+        property_keys = {}
         property_dict = {}
+        deleted_keys = []
+        for sort in sorting_list:
+            for prop_name in sort.get_property_keys():
+                if prop_name in deleted_keys:
+                    continue
+                if prop_name in property_keys:
+                    if property_keys[prop_name] != sort.get_property(prop_name).dtype:
```

Comment:
wouldn't this fail??? Now `property_keys` is initialized as an empty dict!

Reply:
@Dradeliomecus can you add a test for this?

Reply:
Done.